db/migrate/20191111153306_make_playlist_permalinks_unique_within_scope.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'slug'
+
+class Playlist < ActiveRecord::Base
+end
+
+class MakePlaylistPermalinksUniqueWithinScope < ActiveRecord::Migration[6.0]
+  def up
+    Playlist.find_each do |playlist|
+      playlist.reload
```

Comment:
Not blocking, but curious why the AR object needs reloading right out of the gate...

Reply:
Because `find_each` loads a batch and the records in the batch can be stale. It's not very likely because the records are returned in sequence and changes are always made to records with a higher ID. I thought it would be better to be safe than sorry.